cli/config/configfile/file.go

```diff
@@ -183,10 +183,10 @@ func (configFile *ConfigFile) Save() error {
 	if err != nil {
 		return err
 	}
+	defer os.Remove(temp.Name())
```

Comment:
this would also attempt to remove `temp.Name()` in the happy case (i.e., it being succesfully renamed); guess that's not a problem.